README.md

```diff
@@ -21,6 +21,16 @@ You can also use this extension to manage your collection of delicious teas.
 | **TER:**         | https://extensions.typo3.org/extension/tea/        |
 | **Slack:**       | https://typo3.slack.com/channels/qa-best-practices |
 
+## Philosophies
+
+The following is a list of our code and technical philosophies only.
+https://typo3.org/community/teams/best-practices provides an overview on how we work in general.
+
+- Align with TYPO3 core  
```

Comment:
Instead of a (single) bullet point, I propose making this a H3 heading.

Reply:
Done. I've also adjusted to mention the actual folder containing the documentation, as this was merged: https://github.com/FriendsOfTYPO3/tea/pull/1286